alumnium/drivers/appium_driver.py

```diff
@@ -74,7 +74,15 @@ def type(self, id: int, text: str):
 
     @property
     def url(self) -> str:
-        return ""
+        """Get the current URL of the webview context"""
+        current_context = self.driver.current_context
+        for context in self.driver.contexts:
+            if "WEBVIEW" in context:
+                self.driver.switch_to.context(context)
+                break
+        url = self.driver.current_url
```

Comment:
I think this is not available for non-webview contexts (see failures on CI). In this case, we can fallback and return an empty string.